src/nbl/CMakeLists.txt

```diff
@@ -496,29 +496,36 @@ endif()
 target_include_directories(Nabla PUBLIC ${THIRD_PARTY_SOURCE_DIR}/libpng)
 # OpenEXR
 if (_NBL_COMPILE_WITH_OPEN_EXR_)
```

Comment:
for a "good" CI run you need to change `_NBL_COMPILE_WITH_OPEN_EXR_` to be default enabled instead of disabled like now

Reply:
add it here https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/CMakePresets.json#L21C4-L21C25 instead

Reply:
> add it here https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/CMakePresets.json#L21C4-L21C25 instead

yeah thats the proper way to do it